client/src/modules/Globals/Styles/NavBar.css

```diff
@@ -10,7 +10,17 @@
 
   .navbar-searchbar-container {
     margin-left: 21px;
-    margin-right: 70px;
+    margin-right: 150px;
+  }
```

Comment:
Since this class isn't being used with your current implementation, you can delete this. Or, if you decide to implement the changes I suggested above (completely optional) then you can delete `navbar-searchbar-container-not`.